sites/public/src/pages/applications/review/demographics.tsx

```diff
@@ -132,7 +137,11 @@ const ApplicationDemographics = () => {
               <Select
                 id="spokenLanguage"
                 name="spokenLanguage"
-                defaultValue={application.demographics.spokenLanguage}
+                defaultValue={
+                  application.demographics.spokenLanguage?.includes("notListed")
+                    ? application.demographics.spokenLanguage.split(":")[0]
```

Comment:
Should the first value in the ternary always be `notListed`?

Reply:
Yeah, I'll switch to string